f3/granite.go

```diff
@@ -44,6 +45,24 @@ type GMessage struct {
 	Ticket Ticket
 	// Signature by the sender's public key over Instance || Round || Step || Value.
 	Signature []byte
+
+	Evidence AggEvidence
```

Comment:
Can you please remove this from this PR and add it only when it will be used? That should revert dependency changes too.

Reply:
Done